### PR TITLE
feature(wallet-mobile): new tx review for yoroi governance

### PR DESCRIPTION
**Pull request by banklesss**
##### Ticket

YV-40

### CODE REVIEW

File: apps/wallet-mobile/src/features/Staking/Governance/common/helpers.tsx [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  const {styles} = useStyles()
+  const strings = useStrings()
+
+  return (
+    <View style={styles.operation}>
+      <Text style={styles.operationLabel}>{strings.registerStakingKey}</Text>
+
+      <Space width="lg" />
+
+      <Text style={styles.operationValue}>2 ADA</Text>
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Staking/Governance/common/helpers.tsx]: hardcoded

Comment (michalszorad) [reply apps/wallet-mobile/src/features/Staking/Governance/common/helpers.tsx]: Is there anything blocking us from having this dynamic?

Comment (banklesss) [reply apps/wallet-mobile/src/features/Staking/Governance/common/helpers.tsx]: working on it

**Comment by banklesss** [comment]
adding dynamic `keyDeposit `

File: apps/wallet-mobile/src/features/Staking/Governance/common/helpers.tsx (hunk 1 above)
Comment (banklesss) [reply apps/wallet-mobile/src/features/Staking/Governance/common/helpers.tsx]: done

**Comment by banklesss** [comment]
PR ready

**Review by stackchain — APPROVED** [review]

**Pull request merged**